src/test/java/io/strimzi/ValidatingWebhookTest.java

```diff
@@ -54,7 +54,7 @@ public void setup() {
 
     @Test
     public void testEviction() {
-        String podName = "my-cluster-kafka-1";
+        String podName = "my-cluster";
```

Comment:
Why did you changed the name here? The original name was one of the expected ones. The new one is not.